rest-api/pom.xml

```diff
@@ -219,6 +219,13 @@
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>com.github.stefanbirkner</groupId>
+			<artifactId>system-rules</artifactId>
+			<version>1.17.0</version>
+			<scope>test</scope>
+		</dependency>
+
```

Comment:
where are we using this?

Reply:
We aren't. I forgot to remove this after I did some initial testing.